index.js

```diff
@@ -1,29 +1,25 @@
 'use strict';
-var net = require('net');
-var onetime = require('onetime');
+const net = require('net');
 
-module.exports = function (port, opts, cb) {
-	if (typeof opts !== 'object') {
-		cb = opts;
-		opts = {};
-	}
+module.exports = (port, opts) => {
+	opts = Object.assign({}, opts);
```

Comment:
You can always do `opts = opts || {}`. The reason you should use `Object.assign` is when you tend to change properties on the `opts` object later down the road (But I have a feeling you already know this :p)).